### PR TITLE
fixes HOPSWORKS-279 and HOPSWORKS-260 and some css

### PR DESCRIPTION
**Pull request by ErmiasG**
HOPSWORKS-279 - front page not scalling well with some resolutions
HOPSWORKS-260 - Buttons on Jupyter console not showing up
screenshot 1
screenshot from 2017-11-06 16-09-07
screenshot 2
screenshot from 2017-11-06 16-08-49

Changes the home page from  screenshot 1 to screenshot 2 to fix the first issue.
Tested most of the pages for any unintended changes.

### CODE REVIEW

File: hopsworks-web/yo/app/nav.html [hunk 1]
```diff
@@ -1,88 +1,87 @@
...
               <i ng-if="mainCtrl.unreadMessages" class="fa fa-envelope fa-lg" title="Unread messages">
                 <span class="badge" style="background-color: #f89406; font-family: sans-serif; z-index: 100; position: absolute; top: -10px; right: -18px">{{mainCtrl.unreadMessages}}</span>
               </i>
               <i ng-if="!mainCtrl.unreadMessages" class="fa fa-envelope-o fa-lg" title="No unread message"></i>
             </a>
-            <ul uib-dropdown-menu class="dropdown-menu dropdown-messages" >
-              <h3 ng-if="mainCtrl.messages.length" class="text-muted"> Latest messages</h3>
-              <h3 ng-if="!mainCtrl.messages.length" class="text-muted"> Inbox empty.</h3>
-              <li ng-repeat="msg in mainCtrl.messages | limitTo:5">
+            <ul class="dropdown-menu dropdown-messages" >
```
Comment (jimdowling) [inline hopsworks-web/yo/app/nav.html]: I thought that "dropdown-menu" was all gone now with bootstrap 4. It should be uib-dropdown-menu. Are you sure about this? I remember changing all "dropdown-menu" to uib-dropdown when we upgraded to angular 1.6.

File: hopsworks-web/yo/app/views/home.html [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   <div ng-if="mainCtrl.searchTerm.length > 3" class="row">
     <div id="projectSearchResults">
       <div ng-include="'views/search.html'"></div>
     </div>
   </div>
 
-  <div ng-if="mainCtrl.searchTerm.length <= 3" class="row" ng-controller="HomeCtrl as homeCtrl">
-    <div class="col-sm-6 col-md-2 col-lg-2" id="toursBox">
-      <div class="col-md-10 sidebar pull-left" style="min-width: 300px; max-width: 300px">
+  <div class="container content-wrapper" ng-if="mainCtrl.searchTerm.length <= 3" ng-controller="HomeCtrl as homeCtrl">
```
Comment (jimdowling) [inline hopsworks-web/yo/app/views/home.html]: Can we change the searchTerm length to '<=2'. Some people will search with 2 chars.

File: hopsworks-web/yo/app/views/jupyterDashboard.html [hunk 3]
```diff
@@ -384,46 +383,34 @@
               </div>
 
 
+>>>>>>> a6924515f277c59b172a31973161d959c8f05ba9
```
Comment (jimdowling) [inline hopsworks-web/yo/app/views/jupyterDashboard.html]: Are you sure you want this ?!?!?

File: hopsworks-web/yo/app/nav.html (hunk 1 above)
Comment (ErmiasG) [reply hopsworks-web/yo/app/nav.html]: It is still there in bootstrap 4, but we are using bootstrap 3.3.7. And uib is the angular version of bootstrap and we do not need to use it here. When we upgrade to angularjs 1.6 we changed <li class="uib-dropdown"> to <li uib-dropdown>

File: hopsworks-web/yo/app/views/home.html (hunk 2 above)
Comment (ErmiasG) [reply hopsworks-web/yo/app/views/home.html]: This can be done but I do not know if it is appropriate to do it on this jira.

Comment (jimdowling) [reply hopsworks-web/yo/app/views/home.html]: ok. ask mahmoud about it - whether 2 chars is ok.

**Pull request merged**